processing/data_collection/gazette/pipelines.py

```diff
@@ -60,6 +62,8 @@ def process_item(self, item, spider):
             item["source_text"] = self.pdf_source_text(item)
         elif self.is_txt(item["files"][0]["path"]):
             item["source_text"] = self.txt_source_text(item)
+        elif self.is_img(item["files"][0]["path"]):
```

Comment:
I recognize you did a incredible job making this spider work. However, I still think we need to find a way to get the pdf/doc files. Thinking in next project steps, which is make the gazettes files available in single website, it will be better get a pdf version of documents. Those file types are more flexible to be manipulated programmatically.  But I'm not sure if this will be possible. I was studying the requests while interacting with the web site. However, I have no suggestions so far. Sorry...

@sergiomario What do you think? Should we  invest more time trying to get the pdf? Or do you believe using the image files for now are fine?

Reply:
No problems, we only have the website to blame :shrug: . Unfortunately I couldn't find any reliable way to always get a pdf :/

Reply:
Yeah, the web site is quite complex. I`ll try to help you on that.